Libraries/LibJS/Tests/logical-expressions-basic.js

```diff
@@ -17,7 +17,6 @@ try {
     assert(("foo" && true) === true);
     assert((false && "bar") === false);
     assert((true && "bar") === "bar");
-    assert((null && true) === null);
```

Comment:
This should still work, no? Why are we removing it?

Reply:
Oh no worries, it definitely still works. It's just a duplicated test :)

https://github.com/linusg/serenity/blob/d74ad81402c27113b2ed5586bc369703d6be61c4/Libraries/LibJS/Tests/logical-expressions-basic.js#L20

https://github.com/linusg/serenity/blob/d74ad81402c27113b2ed5586bc369703d6be61c4/Libraries/LibJS/Tests/logical-expressions-basic.js#L34

Reply:
Aha! :)